verl/utils/megatron/optimizer.py

```diff
@@ -16,26 +16,22 @@
 import importlib
 from packaging.version import Version
 
-from apex.optimizers import FusedAdam as Adam
-from apex.optimizers import FusedSGD as SGD
-
 from megatron.core.optimizer import OptimizerConfig
-
+from megatron.core.optimizer_param_scheduler import OptimizerParamScheduler
```

Comment:
is this unused import intended?

Reply:
yes. we can remove now and it should be used for LR scheduler config, which I will add in another PR.